UNIXFS.md

```diff
@@ -23,7 +23,7 @@ Draft work and discussion on a specification for the upcoming version 2 of the U
 - [Exporting](#exporting)
 - [Design decision rationale](#design-decision-rationale)
 	- [Metadata](#metadata-1)
-		- [Separate Metadata node](#separate-metadata-node)
+		- [Separate node](#separate-metadata-node)
```

Comment:
```suggestion
		- [Separate Metadata node](#separate-metadata-node)
```

sorry, my first correction was wrong (I did edit it but that might not have propagated)